llvm/lib/SYCLLowerIR/LowerWGScope.cpp

```diff
@@ -818,6 +819,30 @@ PreservedAnalyses SYCLLowerWGScopePass::run(Function &F,
         Allocas.insert(AllocaI);
     }
     for (; I && (I != BB.getTerminator()); I = I->getNextNode()) {
+      if (CallInst *CI = dyn_cast<CallInst>(I)) {
+        if (CI->getCalledFunction()->getName().
+            starts_with(GET_GROUP_PREFIX) &&
```

Comment:
This is the most questionable part of the PR from my perspective. It's done this way to consider possible overloads of get_group_id (depending on dimension template parameter) and potentially touch other functions, that might be affected by the community change, that were not exposed by the regression in the pulldown. @asudarsa @bader WDYT about this line (and the patch in general)?